.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
We should add these to gitignore ?

Reply:
I don't have any hard opinions here.
I just added them because I found setting vscode for jinja with docker a bit annoying.

I do wanna try pycharm though, see if that's better in terms of syntax highlighting, etc.

Reply:
IDE specific settings should be in gitignore i think

Reply:
Fair, Removed!